src/saltext/vault/modules/vault_pki.py

```diff
@@ -0,0 +1,1130 @@
+        Reference of the issuer. Can be issuer id, issuer name or literal ``default``
+        which means default issuer. Defaults to ``default``.
+
+    mount
+        The mount path the PKI backend is mounted to. Defaults to ``pki``.
+
+    """
+    endpoint = f"{mount}/issuer/{ref}"
+
+    try:
+        return vault.query("GET", endpoint, __opts__, __context__, is_unauthd=True)["data"]
+    except vault.VaultNotFoundError:
+        return None
+    except vault.VaultException as err:
+        raise CommandExecutionError(f"{err.__class__}: {err}") from err
+
+
+def update_issuer(
+    ref="default",
+    mount="pki",
+    manual_chain=None,
+    usage=None,
+    aia_urls=None,
+    crl_endpoints=None,
+    ocsp_servers=None,
+):
+    """
+    Update issuer's information.
+
+    `API method docs <https://developer.hashicorp.com/vault/api-docs/secret/pki#update-issuer>`__.
+
+    CLI Example:
+
+    .. code-block:: bash
+
+            salt '*' vault_pki.update_issuer ref usage=["crl-signing"]
+
+    ref
+        Reference of the issuer. Can be issuer id, issuer name or literal ``default``
+        which means default issuer. Defaults to ``default``.
+
+    mount
+        The mount path the PKI backend is mounted to. Defaults to ``pki``.
+
+    manual_chain
+        Chain of issuer references to build this issuer's computed CAChain field from, when non-empty.
+
+    usage
+        Allowed usages for this issuer. Valid options are:
+        ``read-only`` - to allow this issuer to be read; implict; always allowed;
+        ``issuing-certificates`` - to allow this issuer to be used for issuing other certificates;
+        ``crl-signing`` -  to allow this issuer to be used for signing CRLs. This is separate from the CRLSign KeyUsage on the x509 certificate,
+        but this usage cannot be set unless that KeyUsage is allowed on the x509 certificate;
+        ``ocsp-signing`` -  to allow this issuer to be used for signing OCSP responses.
+
+    aia_urls
+        Specifies the URL values for the Issuing Certificate field as an array.
+
+    crl_endpoints
+        Specifies the URL values for the CRL Distribution Points field as an array.
+
+    ocsp_servers
+        pecifies the URL values for the OCSP Servers field as an array.
+
+    """
+    endpoint = f"{mount}/issuer/{ref}"
+
+    payload = {}
+
+    if manual_chain is not None:
+        payload["manual_chain"] = manual_chain
+
+    if usage is not None and len(usage) > 0:
+        payload["usage"] = usage
+
+    if aia_urls is not None:
+        payload["issuing_certificates"] = aia_urls
+
+    if crl_endpoints is not None:
+        payload["crl_distribution_points"] = crl_endpoints
+
+    if ocsp_servers is not None:
+        payload["ocsp_servers"] = ocsp_servers
+
+    try:
+        vault.query(
+            "PATCH",
+            endpoint,
+            __opts__,
+            __context__,
+            payload=payload,
+            add_headers={"Content-Type": "application/merge-patch+json"},
```

Comment:
note: We could implement this in the Vault client instead since afaik it's always required for a `PATCH` request (would need to check again though).

Reply:
Added such change in the client. It's in separate commit, so if it's not okey, will drop it.
All tests are passing (and they should as PATCH is always `json-patch` content-type.